tools/cosmovisor/cmd/cosmovisor/logger.go

```diff
@@ -0,0 +1,20 @@
+package main
+
+import (
+	"os"
+
+	"cosmossdk.io/log"
+	"github.com/rs/zerolog"
+)
+
+func newLoggerFromConfig(nop bool, options ...log.Option) log.Logger {
```

Comment:
I almost feel like it makes sense to return an already configured logger from the `GetConfigFromEnv`. This will reduce code duplication as well. What do you think?